addons/mail/static/src/core/message_model.js

```diff
@@ -114,8 +114,8 @@ export class Message {
         );
     }
 
-    get datetimeShort() {
```

Comment:
removing field/getter in stable is not advised, otherwise it looks ok

Reply:
Let's keep `datetimeShort` for name of getter in the end